include/RunAnalysis.h

```diff
@@ -44,38 +44,49 @@
 #include <regex>
 
 using namespace llvm;
-class MustCall;
-class CalledMethods;
-
+class ProgramFunction;
 namespace rlc_dataflow {
 
+// LLVM analysis results are in the form of structs, this struct will be defined
+// as a result in the CodeAnalyzer struct and will be the return value of the doAnalysis function
+struct ResourceLeakFunctionCallAnalyzerResult {
+    ProgramFunction *programFunctionMustCall;
+    ProgramFunction *programFunctionCalledMethods;
+};
+
+// This represents the analysis that constructs the must call and may called passes
```

Comment:
"may called" = "called methods"? I wouldn't call it the "may called" pass, for two reasons:
* it's worth maintaining consistent names
* the "may" is misleading, since the results of the called methods pass should be underapproximate in the sense that every method in the result must definitely have been called (but some called methods may have been missed)